cmd/gaiad/cmd/root.go

```diff
@@ -101,12 +99,8 @@ func initAppConfig() (string, interface{}) {
 	srvCfg.StateSync.SnapshotKeepRecent = 10
 
 	return params.CustomConfigTemplate, params.CustomAppConfig{
-		Config: *srvCfg,
-		// BypassMinFeeMsgTypes: []string{
```

Comment:
Still don't understand exactly why to move to upgrade handler?

Reply:
sry, i misunderstood that this will change app.toml, so no need to move to upgrade handler.

but when doing `gaia init`, if we want to write the default bypass msg types to app.toml ? if yes, we can uncomment here. line 106-109